### ISSUE TITLE
Dark mode with IDL blocks and Notes

### ISSUE THREAD

**Issue by rtoy** [tabatkins/bikeshed#1797]
In light mode, IDL blocks have a blue left border and a light blue background.  Similarly, notes have a green left border with a light green background.  In dark mode, backgrounds are basically invisible.  Not sure if that's intentional or not.  If it is intentional, feel free to close this issue.

**Comment by tabatkins** [comment]
Intentional - I couldn't find good colors for them, as finding good-looking dark shades is a lot more challenging than pastels.

That said, suggestions welcome.

**Comment by rtoy** [comment]
Yeah, it is hard. Playing around with the color picker, I think this isn't a terrible blue background for IDL blocks: 12095f. This is an ok color for Notes:  005a00. The current colors for the IDL blocks makes it hard to find a nice color where you can still see the IDL colors clearly.

I guess no matter what color you pick, someone is not going to like them.

**Comment by rtoy** [comment]
Also Examples sections don't have a background.  I couldn't find a yellow hue that I thought looked reasonable.  They're either too yellow and loses contrast with the text or they're so dark as to be barely visible on the black background.
